package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "cd frontend && react-scripts start",
     "build": "cd frontend && react-scripts build",
-    "test": "jest",
+    "test": "jest --testPathIgnorePatterns prereq_loader",
```

Comment:
I think DJ merged in the commit to address the bug you found. So we should probably remove this before getting it merged in.

Reply:
yep!! Will do, thanks!